app/models/user.rb

```diff
@@ -116,6 +116,17 @@ def send_devise_notification(notification, *args)
 		message = devise_mailer.delay.send(notification, self, *args)
 	end
 
+	# override devise class method send_reset_password_instructions to limit to 1 request / 5 min
+	def self.send_reset_password_instructions(attributes={})
+		recoverable = find_or_initialize_with_errors(reset_password_keys, attributes, :not_found)
+		if recoverable.persisted? && (recoverable.reset_password_sent_at.nil? || Time.now > recoverable.reset_password_sent_at + 5.minutes)
+      recoverable.send_reset_password_instructions
+			return recoverable
+		end
+		recoverable.errors.add(:user, "can't reset password because a request was just sent")
```

Comment:
As written, this seems to add the error for "can't reset password because a request was just sent" any time the recoverable is not persisted. Based on how `find_or_initialize_with_errors` works, ` recoverable` would not be persisted if there's no user with the email passed in attributes. We shouldn't be adding this error to those users.

Reply:
Ohh good point good point. Thanks for catching that

Reply:
Just pushed a commit with this change